package.json

```diff
@@ -10,14 +10,15 @@
     "access": "public"
   },
   "files": [
+    "facet-filter-sort.serge.json",
     "custom-elements.json",
-    "/components"
+    "/components",
+    "/lang"
   ],
   "scripts": {
-    "lint": "npm run lint:eslint && npm run lint:lit && npm run lint:polymer && npm run lint:style",
+    "lint": "npm run lint:eslint && npm run lint:lit && npm run lint:style",
     "lint:eslint": "eslint . --ext .js,.html",
     "lint:lit": "lit-analyzer \"{components, test, demo}/**/*.js\" --strict --rules.no-unknown-tag-name off",
-    "lint:polymer": "polymer lint -i \"test/**/*.js\" \"demo/**/*.js\" \"components/**/*.js\"",
```

Comment:
Polymer linting hated our Lit dynamic import, so it was easier to just disconnect it at this point.